openapi/azure.sqldb/openapi.yml

```diff
@@ -0,0 +1,1171 @@
+
+             * 400 RequiredBackupIsNotLastRestored - Migration cannot be completed because provided backup file name is not the name of the last backup file that is restored.
+
+             * 400 IncompatiblePhysicalLayoutTooFewDataFiles - Database backup contains incompatible physical layout. No data files are found in the backup.
+
+             * 400 IncompatiblePhysicalLayoutNonOnlineDataFilesExist - Database backup contains incompatible physical layout. Non-online data files exist.
+
+             * 400 IncompatiblePhysicalLayoutWrongNumberOfLogFiles - Database backup contains incompatible physical layout. Multiple log files are not supported.
+
+             * 400 IncompatiblePhysicalLayoutNonOnlineLogFilesExist - Database backup contains incompatible physical layout. Non-online log files exist.
+
+             * 400 IncompatiblePhysicalLayoutTooManyNonDataLogFiles - Database backup contains incompatible physical layout. Too many non-data/log files.
+
+             * 400 BackupSetNotFound - "No backups were found to restore the database. Please contact support to restore the database.
+
+             * 400 FullBackupNotFound - Full backup can not be found.
+
+             * 400 RestoreFromStripedBackupsNotEnabled - Restoring from striped backups is not supported.
+
+             * 400 BrokenRestorePlanNoFullBackup - The restore plan is broken because there is no full backup.
+
+             * 400 BrokenRestorePlanWrongLogBackupLSN - The restore plan is broken because firstLsn of current log backup is not <= lastLsn of next log backup.
+
+             * 400 BrokenRestorePlanWrongDiffBackupLSN - The restore plan is broken because firstLsn of diff backup is not >= firstLsn of full backup.
+
+             * 400 BrokenRestorePlanGapInLogBackups - The restore plan is broken because firstLsn of current log backup is not equal to lastLsn of prev log backup.
+
+             * 400 ManagedInstanceStorageLimitHit - The managed instance has reached its storage limit.
+          content: {}
+      x-ms-examples:
+        Completes a managed database restore:
+          parameters:
+            api-version: 2017-03-01-preview
+            locationName: southeastasia
+            operationId: 111111111-2222-8888-5555-777777777777/completeRestore
+            parameters:
+              lastBackupName: testdb1_log4
+            subscriptionId: 00000000-1111-2222-3333-444444444444
+          responses:
+            200: {}
+            202: {}
+      x-ms-long-running-operation: true
+      x-codegen-request-body-name: parameters
+  ? /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/managedInstances/{managedInstanceName}/databases
+  : get:
+      tags:
+      - ManagedDatabases
+      description: Gets a list of managed databases.
+      operationId: ManagedDatabases_ListByInstance
```

Comment:
Please start the operation ID with a verb and apply where necessary in this file so that the remote operations in the client.bal will have meaningful names

Reply:
Hi, Did the suggested changes. Please check.
/Thanks